src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/SampleTableActions/index.tsx

```diff
@@ -77,6 +80,17 @@ const SampleTableActions = ({
         onEditSelected={openEditSampleModal}
         data-test-id="sample-page-more-action-btn"
       />
+      <Link href={ROUTES.UPLOAD_STEP1} passHref>
```

Comment:
Ya, old and new tables.  It's awkward because the previous implementation used next link, similar to this one, but I ended up removing the next link because it wasn't passing the href properly (probably because the sds component doesn't actually take that field).  It's possible this link will need to be updated to also not use next link.